web-registry/src/pages/Projects/hooks/useProjectsSellOrders.utils.tsx

```diff
@@ -38,3 +39,18 @@ export const getPurchaseInfo = (
     },
   };
 };
+
+export const sortProjectsBySellOrdersAvailability =
+  (sellOrders: SellOrderInfo[]) =>
+  (projectA: ProjectInfo, projectB: ProjectInfo) => {
+    const ordersForProjectA = sellOrders.filter(order =>
+      order.batchDenom.startsWith(projectA?.id),
```

Comment:
```suggestion
      order.batchDenom.startsWith(`${projectA?.id}-`),
```
An edge case, but I think we need to do this so it doesn't pick up other project IDs, for example C01-111 vs C01-1111.